design/src/components/Glogg/Glogg.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Champagne from "/Users/durreshewarmalik/repos/HTML_CSS/GV_Design-Sprint/design/src/assets/img/champagne.jpg";
```

Comment:
These paths are relative to your own computer and will not work for others. Change the path to something generic, like:
`./src/assets/img/champagne.jpg`

Same comment goes for lines 3-5.

Reply:
../../assets/img/champagne.jpg

done